tests/client/core/api/test_api.js

```diff
@@ -7,18 +7,26 @@ import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+const defaultHeaders = {
+  'Content-Type': 'application/json',
+};
+
+function generateHeaders(headerData = defaultHeaders) {
+  return {
+    get: (name) => headerData[name],
```

Comment:
Our tests do not all pass in Safari (and it doesn't really work well with karma for some reason) but it looks like this updated test using `Response` passes.